scope_ascom.cpp

```diff
@@ -866,6 +866,18 @@ bool ScopeASCOM::GetGuideRates(double *pRAGuideRate, double *pDecGuideRate)
         }
 
         *pRAGuideRate = vRes.dblVal;
+
+        if (!ValidGuideRates(*pRAGuideRate, *pDecGuideRate))
+        {
+            if (!m_bogusGuideRatesFlagged)
+            {
+                pFrame->Alert(_("Mount is reporting invalid guide speeds, some guiding functions including PPEC will be impaired. Contact vendor for support."),

```

Comment:
hmm, maybe we should blame the ascom driver more directly in the error message?  I believe there are still some mounts out there where the ascom driver provider is not the mount vendor. Tak is one example, there may be more.
```suggestion
                pFrame->Alert(_("The mount's ASCOM driver is reporting invalid guide speeds. Some guiding functions including PPEC will be impaired. Contact the ASCOM driver provider or mount vendor for support."),
```